tools/packaging/static-build/cache_components_main.sh

```diff
@@ -48,6 +49,19 @@ cache_qemu_artifacts() {
 	create_cache_asset "${qemu_tarball_name}" "${current_qemu_version}-${qemu_sha}" "${current_qemu_image}"
 }
 
+cache_rootfs_artifacts() {
+	local osbuilder_last_commit="$(get_last_modification "${repo_root_dir}/tools/osbuilder")"
```

Comment:
In the install_image logic you have `sed s/-dirty//` as the rootfs artefacts are generated in the osbuilder directory. I think it is needed here as well to match them up?

Reply:
Hmm.  I actually can remove the `-dirty` everywhere, as this is only needed for the case we generate the root hashes.
Thanks for spotting this!